btc_parser/sources/output.move

```diff
@@ -27,3 +53,73 @@ public fun amount(output: &Output): u64 {
 public fun script_pubkey(output: &Output): vector<u8> {
     output.script_pubkey
 }
+
+public fun is_P2PHK(output: &Output): bool {
+    let script = output.script_pubkey();
+
+    script.length() == 25 &&
+		script[0] == OP_DUP &&
+		script[1] == OP_HASH160 &&
+		script[2] == OP_DATA_20 &&
+		script[23] == OP_EQUALVERIFY &&
+		script[24] == OP_CHECKSIG
+}
+
+public fun is_op_return(output: &Output): bool {
+    let script = output.script_pubkey;
+    script.length() > 0 && script[0] == OP_RETURN
+}
+
+public fun is_P2WPHK(output: &Output): bool {
+    let script = output.script_pubkey;
+    script.length() == 22 &&
+        script[0] == OP_0 &&
+        script[1] == OP_DATA_20
+}
+
+// TODO: add support script addresses.
```

Comment:
lets create tasks for it so we dont miss it

Reply:
created